tests/python_tests/conftest.py

```diff
@@ -3,6 +3,7 @@
 import shutil
 import logging
 from utils.constants import get_ov_cache_models_dir
+from utils.tokenizers import model_tmp_path  # NOQA: F401
```

Comment:
I missed this change in the original PR. I always struggled to follow the logic. That's pretty strange to me to enable style checks and then work around tool's bugs with comments. Not enabling the check at all would be easier at the cost of random unused imports. Style checkers are supposed to help to write readable the code, not make it harder.

Can you explain?

Reply:
This import is not unused, it is just the way pytest handles fixtures. Moved the `model_tmp_path` import back to `test_tokenizers`